Old scrollkeeper-style docs fail to install.

--- origsrc/gnome-games-2.26.2/gnobots2/help/Makefile.am	2009-05-18 19:15:21.000000000 -0500
+++ src/gnome-games-2.26.2/gnobots2/help/Makefile.am	2009-06-09 14:09:41.800881300 -0500
@@ -2,7 +2,7 @@
 # remove your language code from SUBDIRS here and from AC_CONFIG_FILES
 # in configure.in once you've migrated.
 
-SUBDIRS = da
+# SUBDIRS = da
 
 # GNOME doc utils
 
--- origsrc/gnome-games-2.26.2/gtali/help/Makefile.am	2009-05-18 19:15:21.000000000 -0500
+++ src/gnome-games-2.26.2/gtali/help/Makefile.am	2009-06-09 13:51:13.767950800 -0500
@@ -2,7 +2,7 @@
 # remove your language code from SUBDIRS here and from AC_CONFIG_FILES
 # in configure.in once you've migrated.
 
-SUBDIRS = da
+#SUBDIRS = da
 
 # GNOME doc utils
 
