--- origsrc/gnome-games-2.27.92/libgames-support/Makefile.am	2009-09-07 16:56:55.000000000 -0500
+++ src/gnome-games-2.27.92/libgames-support/Makefile.am	2009-09-08 14:09:18.737278400 -0500
@@ -358,6 +358,7 @@ pkglib_LTLIBRARIES += libgames-support-g
 libgames_support_gi_la_SOURCES = $(libgames_support_la_SOURCES) $(nodist_libgames_support_la_SOURCES)
 libgames_support_gi_la_CPPFLAGS = $(libgames_support_la_CPPFLAGS) -DLIBGAMES_SUPPORT_GI
 libgames_support_gi_la_CFLAGS = $(libgames_support_la_CFLAGS)
+libgames_support_gi_la_LDFLAGS = -no-undefined
 libgames_support_gi_la_LIBADD = $(libgames_support_la_LIBADD)
 
 BUILT_GIRSOURCES = GnomeGamesSupport-1.0.gir GGZ-1.0.gir
@@ -371,7 +372,7 @@ GnomeGamesSupport-1.0.gir: $(INTROSPECTI
 	--include=Cogl-1.0 \
 	--include=Gtk-2.0 \
 	--include=GGZ-1.0 \
-	--library=games-support-gi \
+	--library=libgames-support-gi.la \
 	--libtool="$(LIBTOOL)" \
 	--output $@ \
 	--pkg gobject-2.0 \
