--- origsrc/gnome-games-2.32.1/lightsoff/src/main.js	2010-04-10 00:01:16.000000000 -0500
+++ src/gnome-games-2.32.1/lightsoff/src/main.js	2011-03-10 23:53:29.306731000 -0600
@@ -1,5 +1,9 @@
 #!/usr/bin/env seed
 
+imports.gi.versions.Clutter = "1.0";
+imports.gi.versions.Gtk = "2.0";
+imports.gi.versions.GtkClutter = "0.10";
+
 GtkClutter = imports.gi.GtkClutter;
 Clutter = imports.gi.Clutter;
 Gtk = imports.gi.Gtk;
--- origsrc/gnome-games-2.32.1/swell-foop/src/main.js	2010-04-10 00:01:17.000000000 -0500
+++ src/gnome-games-2.32.1/swell-foop/src/main.js	2011-03-10 23:54:15.124010900 -0600
@@ -4,6 +4,8 @@ var tile_size = 50;
 var offset = tile_size/2;
 
 imports.gi.versions.Clutter = "1.0";
+imports.gi.versions.Gtk = "2.0";
+imports.gi.versions.GtkClutter = "0.10";
 
 Gtk = imports.gi.Gtk;
 GtkClutter = imports.gi.GtkClutter;
