--- origsrc/gnome-games-3.0.0/glchess/src/glchess.c	2011-04-04 11:10:41.000000000 -0500
+++ src/gnome-games-3.0.0/glchess/src/glchess.c	2011-04-17 06:32:33.110435200 -0500
@@ -3233,7 +3233,7 @@ gboolean G_MODULE_EXPORT preferences_del
 	g_return_val_if_fail (self != NULL, FALSE);
 	g_return_val_if_fail (widget != NULL, FALSE);
 	g_return_val_if_fail (event != NULL, FALSE);
-	G_MODULE_EXPORT preferences_response_cb (widget, (gint) GTK_RESPONSE_CANCEL, self);
+	preferences_response_cb (widget, (gint) GTK_RESPONSE_CANCEL, self);
 	result = TRUE;
 	return result;
 }
