--- origsrc/gnome-games-3.0.0/libgames-support/Makefile.am	2011-04-17 05:39:12.961397200 -0500
+++ src/gnome-games-3.0.0/libgames-support/Makefile.am	2011-04-17 06:14:05.935108400 -0500
@@ -208,7 +208,7 @@ GnomeGamesSupport-1.0.gir: $(INTROSPECTI
 	--include=Clutter-1.0 \
 	--include=Cogl-1.0 \
 	--include=Gtk-$(GTK_API_VERSION) \
-	--library=games-support-gi \
+	--library=libgames-support-gi.la \
 	--libtool="$(LIBTOOL)" \
 	--output $@ \
 	--pkg gobject-2.0 \
@@ -218,7 +218,7 @@
 	--warn-all \
 	-I$(top_srcdir) \
 	-I$(top_builddir) \
-	$(filter games%,$^)
+	$(filter $(srcdir)/games-%.c $(srcdir)/games-%.h,$^)
 
 girdir = $(pkglibdir)
 gir_DATA = $(BUILT_GIRSOURCES)
