--- origsrc/gnome-games-2.22.3/configure.in	2008-06-30 01:09:36.000000000 -0500
+++ src/gnome-games-2.22.3/configure.in	2008-07-10 14:27:02.046875000 -0500
@@ -685,7 +685,7 @@
 AC_CHECK_HEADERS([fcntl.h limits.h sys/time.h time.h \
                   unistd.h], [],
                  [AC_MSG_ERROR([cannot find required header file])])
-AC_CHECK_HEADERS([netdb.h sys/socket.h winsock2.h])
+AC_CHECK_HEADERS([netdb.h sys/socket.h])
 AC_HEADER_SYS_WAIT
 
 
@@ -723,6 +723,7 @@
 case "$host" in
   *-*-mingw*)
     os_win32=yes
+    AC_CHECK_HEADERS([winsock2.h])
     ;;
   *)
     os_win32=no
--- origsrc/gnome-games-2.22.3/dependencies/ggz-gtk/Makefile.am	2008-06-29 23:22:50.000000000 -0500
+++ src/gnome-games-2.22.3/dependencies/ggz-gtk/Makefile.am	2008-07-10 14:27:02.078125000 -0500
@@ -71,7 +71,7 @@
 			xtext.h \
 			xtext-ggz.h
 
-libggz_gtk_la_LIBADD = $(GTK_LIBS) $(LDADD) $(LIBS) $(INTLLIBS) \
+libggz_gtk_la_LIBADD = $(GNOME_GAMES_LIBS) $(LDADD) $(LIBS) $(INTLLIBS) \
 			$(LIB_GGZCORE) $(LIB_GGZ)
 libggz_gtk_la_LDFLAGS = $(GGZCORE_LDFLAGS) $(LIBGGZ_LDFLAGS) -version-info 0:0:0
 
--- origsrc/gnome-games-2.22.3/gnect/data/Makefile.am	2008-06-30 00:30:29.000000000 -0500
+++ src/gnome-games-2.22.3/gnect/data/Makefile.am	2008-07-10 14:27:02.093750000 -0500
@@ -14,9 +14,9 @@
 if GGZ_SERVER
 ggzroom_in_files = gnect.room.in
 ggzroom_DATA = $(ggzroom_in_files:.room.in=.room)
-ggzroomdir = ${prefix}/etc/ggzd/rooms/
+ggzroomdir = $(sysconfdir)/ggzd/rooms/
 
-ggzserver_dscdir = ${prefix}/etc/ggzd/games/
+ggzserver_dscdir = $(sysconfdir)/ggzd/games/
 ggzserver_dsc_in_files = gnect-server.dsc.in
 ggzserver_dsc_DATA = $(ggzserver_dsc_in_files:.dsc.in=.dsc)
 endif # GGZ_SERVER
--- origsrc/gnome-games-2.22.3/gnect/src/Makefile.am	2008-06-30 00:30:29.000000000 -0500
+++ src/gnome-games-2.22.3/gnect/src/Makefile.am	2008-07-10 14:27:02.109375000 -0500
@@ -73,5 +73,5 @@
 	../../libgames-support/libgames-support.la         \
 	$(GNOME_GAMES_LIBS)     \
 	$(INTLLIBS) \
-	$(LIB_GGZMOD) \
-	$(LIB_GGZ_GTK)
+	$(LIB_GGZ_GTK) \
+	$(LIB_GGZMOD)
--- origsrc/gnome-games-2.22.3/gnibbles/Makefile.am	2008-06-30 00:30:29.000000000 -0500
+++ src/gnome-games-2.22.3/gnibbles/Makefile.am	2008-07-10 14:27:02.140625000 -0500
@@ -58,8 +58,8 @@
 	$(GGZDMOD_INCLUDES)
 
 gnibbles_LDADD += \
-	$(LIB_GGZMOD) \
-	$(LIB_GGZ_GTK)
+	$(LIB_GGZ_GTK) \
+	$(LIB_GGZMOD)
 endif
 
 pixmapdir = $(datadir)/pixmaps/gnibbles
@@ -132,9 +132,9 @@
 
 ggzroom_in_files = gnibbles.room.in
 ggzroom_DATA = $(ggzroom_in_files:.room.in=.room)
-ggzroomdir = ${prefix}/etc/ggzd/rooms/
+ggzroomdir = $(sysconfdir)/ggzd/rooms/
 
-ggzserver_dscdir = ${prefix}/etc/ggzd/games/
+ggzserver_dscdir = $(sysconfdir)/ggzd/games/
 ggzserver_dsc_in_files = gnibbles-server.dsc.in
 ggzserver_dsc_DATA = $(ggzserver_dsc_in_files:.dsc.in=.dsc)
 endif # GGZ_SERVER
--- origsrc/gnome-games-2.22.3/iagno/Makefile.am	2008-06-30 00:30:29.000000000 -0500
+++ src/gnome-games-2.22.3/iagno/Makefile.am	2008-07-10 14:27:02.703125000 -0500
@@ -47,8 +47,8 @@
 	$(top_builddir)/libgames-support/libgames-support.la \
 	$(GNOME_GAMES_LIBS) \
 	$(INTLLIBS) \
-	$(LIB_GGZMOD) \
-	$(LIB_GGZ_GTK)
+	$(LIB_GGZ_GTK) \
+	$(LIB_GGZMOD)
 
 pixmap_DATA = \
 	classic.png \
@@ -65,9 +65,9 @@
 if GGZ_SERVER
 ggzroom_in_files = iagno.room.in
 ggzroom_DATA = $(ggzroom_in_files:.room.in=.room)
-ggzroomdir = ${prefix}/etc/ggzd/rooms/
+ggzroomdir = $(sysconfdir)/ggzd/rooms/
 
-ggzserver_dscdir = ${prefix}/etc/ggzd/games/
+ggzserver_dscdir = $(sysconfdir)/ggzd/games/
 ggzserver_dsc_in_files = iagno-server.dsc.in
 ggzserver_dsc_DATA = $(ggzserver_dsc_in_files:.dsc.in=.dsc)
 endif # GGZ_SERVER
--- origsrc/gnome-games-2.22.3/m4/platform.m4	2008-06-29 23:22:51.000000000 -0500
+++ src/gnome-games-2.22.3/m4/platform.m4	2008-07-10 14:27:02.718750000 -0500
@@ -65,9 +65,8 @@
   if test "$MINGW32" = "yes"; then
     LDFLAGS="$LDFLAGS -no-undefined"
     LDADD="$LDADD -lws2_32"
+    AC_CHECK_HEADERS([winsock2.h])
   fi
-
-  AC_CHECK_HEADERS([winsock2.h])
 ])
 
 dnl ------------------------------------------------------------------------
